storage/storage-mq/src/storage-content/postgresStorageContentRepository.ts

```diff
@@ -43,28 +43,26 @@ export class PostgresStorageContentRepository implements StorageContentRepositor
   async existsTable (tableIdentifier: string): Promise<boolean> {
     const resultSet =
       await this.postgresRepository.executeQuery(EXISTS_TABLE_STATEMENT(POSTGRES_SCHEMA, tableIdentifier), [])
-    const tableExists = !!resultSet.rows[0].to_regclass
-    return Promise.resolve(tableExists)
+    return !!resultSet.rows[0].to_regclass
```

Comment:
I would readd the explanatory variabale `tableExists`. Using these sort of unnecessary variables on the first glance with speaking name improves the readability of the code quite heavily in my opinion. Probably better named `foundTableWithIdentifier` or something similar since `tableExists` is duplicate with method name

Reply:
Ok, I have readded the variable.